client.cfg

```diff
@@ -20,23 +20,24 @@
 ##
 ## Client variable macro for your printer.cfg
 #[gcode_macro _CLIENT_VARIABLE]
-#variable_use_custom_pos   : False ; use custom park coordinates for x,y [True/False]
-#variable_custom_park_x    : 0.0   ; custom x position; value must be within your defined min and max of X
-#variable_custom_park_y    : 0.0   ; custom y position; value must be within your defined min and max of Y
-#variable_custom_park_dz   : 2.0   ; custom dz value; the value in mm to lift the nozzle when move to park position
-#variable_retract          : 1.0   ; the value to retract while PAUSE
-#variable_cancel_retract   : 5.0   ; the value to retract while CANCEL_PRINT
-#variable_speed_retract    : 35.0  ; retract speed in mm/s
-#variable_unretract        : 1.0   ; the value to unretract while RESUME
-#variable_speed_unretract  : 35.0  ; unretract speed in mm/s
-#variable_speed_hop        : 15.0  ; z move speed in mm/s
-#variable_speed_move       : 100.0 ; move speed in mm/s
-#variable_park_at_cancel   : False ; allow to move the toolhead to park while execute CANCEL_PRINT [True/False]
-#variable_park_at_cancel_x : None  ; different park position during CANCEL_PRINT [None/Position as Float]; park_at_cancel must be True
-#variable_park_at_cancel_y : None  ; different park position during CANCEL_PRINT [None/Position as Float]; park_at_cancel must be True
+#variable_use_custom_pos         : False ; use custom park coordinates for x,y [True/False]
+#variable_custom_park_x          : 0.0   ; custom x position; value must be within your defined min and max of X
+#variable_custom_park_y          : 0.0   ; custom y position; value must be within your defined min and max of Y
+#variable_custom_park_dz         : 2.0   ; custom dz value; the value in mm to lift the nozzle when move to park position
+#variable_retract                : 1.0   ; the value to retract while PAUSE
+#variable_cancel_retract         : 5.0   ; the value to retract while CANCEL_PRINT
+#variable_speed_retract          : 35.0  ; retract speed in mm/s
+#variable_unretract              : 1.0   ; the value to unretract while RESUME
+#variable_speed_unretract        : 35.0  ; unretract speed in mm/s
+#variable_speed_hop              : 15.0  ; z move speed in mm/s
+#variable_speed_move             : 100.0 ; move speed in mm/s
+#variable_park_at_cancel         : False ; allow to move the toolhead to park while execute CANCEL_PRINT [True/False]
```

Comment:
I do not like that you splitt the poark canclel paramter. Please place your parameter before line 38